src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -75,7 +124,11 @@ class LambdaTest extends ExternalServicesTestUtils with MockitoSugar {
         MockLambda(
           convertFolderIdsAndRowsToListOfIoRows(folderIdsAndRows),
           entitiesWithSourceIdReturnValue =
-            defaultEntitiesWithSourceIdReturnValues.take(1) ++ List(responseWithNoEntity, responseWithNoEntity)
+            defaultEntitiesWithSourceIdReturnValues.take(1) ++ List(responseWithNoEntity, responseWithNoEntity),
+          addEntityReturnValues = List(
+            IO(structuralObjects(1).head.ref),
+            IO(structuralObjects(2).head.ref)
```

Comment:
There's quite a few of these `IO(structuralObjects(idx).head.ref)` Is it worth putting in a method like `def idForIdx(idx: Int): IO[UUID] = IO(structuralObjects(idx).head.ref)` (with different names)?